case-studies/policy/cli.md

```diff
@@ -43,81 +45,148 @@ The create command is used to create resources by filename.
     * GlobalNetworkPolicy(or gnp)
 
 Usage:
-  bbfwcli create [resourceType] [flags]
+  bbfw create [resourceType] [flags]
 
 Examples:
   # Create a global network policy
-  bbfwcli create gnp policy.yaml
+  bbfw create gnp -f policy.yaml
 
   # Create many global network policy
-  bbfwcli create gnp policy1.yaml policy2.yaml
+  bbfw create gnp -f policy1.yaml -f policy2.yaml
 
 Flags:
   -f, --file stringArray   file to read
   -h, --help               help for create
 ```
 
+## List resource
+```shell
+List resource
+
+Usage:
+  bbfw list [flags]
+
+Examples:
+  # List global network policy
+  bbfw list gns
+
+  # List global network policy
+  bbfw list gnp
+
+  # List global network policy with order
+  bbfw list gnp --isOrder
+
+  # List host endpoint
+  bbfw list hep
```

Comment:
"hep" is not commonly used to represent "host endpoints".

Reply:
hep is abbreviation of hostendpoint. You can use bbfw list hostendpoint